app/post/[id]/page.tsx

```diff
@@ -19,20 +21,24 @@ export default async function Post({ params }: { params: { id: string } }) {
     // TODO: error handling
     const hydratedCreateReply = createReply.bind(null, post.id);
     return (
-        <PostComponent message={post} variant={PostVariant.DETAIL_VIEW}>
-            {user ? (
-                <PostForm
-                    user={user}
-                    messageVariant={MessageVariant.REPLY}
-                    onCreate={hydratedCreateReply}
-                />
-            ) : (
-                <div className="flex flex-row items-center gap-xs py-l">
-                    <LoginButton session={null} loginLabel="Logge dich jetzt ein" />
-                    <span>um einen Kommentar zu verfassen.</span>
-                </div>
-            )}
-            <ReplyContainer paginatedReplies={replies} />
-        </PostComponent>
+        <div className="flex w-full flex-col px-m md:w-[680px] md:px-0">
+            <Suspense fallback={<LoadingPostReplies />}>
```

Comment:
same here with suspense and loading.tsx file - as I understood it, we only need one of each, or am I wrong?